lib/trento/activity_logging/activity_catalog.ex

```diff
@@ -175,6 +175,7 @@ defmodule Trento.ActivityLog.ActivityCatalog do
       {TrentoWeb.V1.SettingsController, :update_activity_log_settings} =>
         {:activity_log_settings_update, 200},
       {TrentoWeb.V1.HostController, :request_operation} => {:host_operation_requested, 202},
+      {TrentoWeb.V1.HostController, :delete} => {:host_cleanup_requested, 204},
```

Comment:
Good one. I think we need to also register
- `SapSystemController, :delete_application_instance`
- `DatabaseController, :delete_database_instance`